test/integration_test/kubevirt_test.go

```diff
@@ -246,3 +251,67 @@ func createDatadiskTemplates(t *testing.T) error {
 	}
 	return nil
 }
+
+func kubevirtVMScaledDeployAndValidate(t *testing.T, instanceID, appKey string, scale int) []*scheduler.Context {
```

Comment:
I think this function should take multiple appKeys since there are tests that need that. I still think it would be better to add the scale factor to the existing function instead of creating this new function which lacks the ability to deploy multiple apps. 

If the scale is not needed, it is easy to pass 1 as the scale factor.

Reply:
Fixed.